src/patternfly/components/ModalBox/modal-box.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="pf-c-modal-box {{ pf-c-modal-box--modifier }}" 
+  role="dialog" 
+  aria-labelledby="{{ pf-c-modal-box__label }}"
```

Comment:
I'm not seeing a value inserted for this attribute in the html example. I'm also not seeing where a value for `id` is set on the title, which is what the value should be for `aria-labelledby` in this case.

One idea is to have a single `{{ pf-c-modal-box--id }}` property for the modal-box.hbs handlebar, and that id value is then used to construct all other id values, like:
```
<div class="pf-c-modal-box__header {{ pf-c-modal-box__header--modifier }}" id="{{ pf-c-modal-box--id }}-header">
  {{> @partial-block}}
</div>
```
and then `aria-labelledby="{{ pf-c-modal-box--id }}-header"`

I think that would work?

Reply:
Are you not seeing the value on the demo? It doesn't show up on the modal-box component because there's nothing to add.

Reply:
Doh, nevermind. I see that it's working as expected in the demo.

Reply:
Are you saying you'd always label it by the modal-box's header? I was assuming that it would have to be just the part that was descriptive, like title. The header also contains the close button and could technically be anything.